### ISSUE TITLE
You must add at least 1 account - Cannot use app for datamining historic data without actually scanning

### ISSUE THREAD

**Issue by SoonDead** [mchristopher/PokemonGo-DesktopMap#1287]
TL;DR: Is there a way to configure the app, so I can use the full stats page but not the scanner?
## Motivation:

I'm trying to get some people into locating nests from historic data. This map is much easier to install than PokemonGoMap and is aimed to be newbie friendly, so it would be an ideal choice for people unable or unwilling to install and configure stuff like python, environment variables, etc. I would recommend them to install this, copy my pogom.db over theirs and view the stats, but this map cannot be started without authenticating to niantic and actually start scanning.
## Possible Solution:

Let the app start without any accounts. Just like when you use the `-os` (only server) flag in PokemonGoMap. This can be either done by removing the check for accounts, or by adding a "Start server, but do not scan" button, which ignores all account and authentication stuff and starts up in only-server mode.

I know that since noone develops this app any more this issue will get ignored, but I'll try anyway.

**Comment by fusenuk** [comment]
You could try editing the main.js file once it is installed and add the -os flag to see what happens when you click 'start server'.

The file to edit would be this one - https://github.com/mchristopher/PokemonGo-DesktopMap/blob/master/app/main.js

Which on Windows is *user*\AppData\Local\Pokemon\app-0.3.4\resources\app\main.js

You'd need to add a line after line number 211 with 

 '--os',

save then run the program. May not actually work but you could check.

**Comment by SoonDead** [comment]
Actually editing login.html and commenting out the check `if (userAccounts.length <= 0) return false;` helps getting further, but now I'm stuck at "Loading, please wait...".

Adding '--os' does not seem to have any effect.

I would like to see a console output / log from the python background processes to find the error but I wasn't able to dig them out yet..
